docs/VnVPlan/VnVPlan.tex

```diff
@@ -549,8 +549,318 @@ \subsection{Tests for Nonfunctional Requirements}
+  the code and interface and ensure components are implemented as they are
+  defined in the SRS.
+  \item[Accessibility Check:] A developer will review if accessibility tools
+  exist or not within the application. Important tools are screen readers and
+  alt text for images. Furthermore, the developer must check for sufficient
+  colour contrast between components.
+\end{description}
+
+
+\begin{enumerate}
+
+\item{NFR-UI1}
+
+NFR: LF-AR1, LF-AR2, LF-AR3, LF-SR1, LF-SR2, LF-SR3, LF-SR4
+
+Type: Manual, Front End Review
+					
+Initial State: Application on home page, ready for use
+					
+Input/Condition: Tester engagement
+					
+Output/Result: List of all identified inconsistencies that are found that do not adhere to SRS defined requirements.
+					
+How test will be performed: 
+\begin{enumerate}
+  \item A developer will review the code to make sure the look and feels requirements were implemented
+  \item They will then look at the application as well to ensure that the implementation was made correctly
+\end{enumerate}
+					
+\item{NFR-UI2}
+
+NFR: CR-SC3
+
+Type: Manual, Accessibility Check
+					
+Initial State: Application on home page, ready for use
+					
+Input: Tester engagement
+					
+Output: List of missing accessibility considerations that have been overlooked
+					
+How test will be performed: 
+\begin{enumerate}
+  \item The developer will review the code and application to ensure that accessibility considerations have been implemented
+  \item They will verify that all images have alt text
+  \item The developer will note down any missing implementations of accessibility requirements
+\end{enumerate}
+
+\end{enumerate}
 
 \subsubsection{Usability and Humanity Requirements}
+These requirements are intended to build a foundation for the user's experience
+on the application. Thus, the only real way to get feedback the user experience
+is to have someone use the application and provide feedback.
+
+\begin{description}
+  \item[User Test Demo:] We will engage a user to test the application, bringing
+  them through a new user process, and ask for feedback on specific areas
+  relevant to the requirements that were set.
+\end{description}
+
+
+\begin{enumerate}
+
+  \item{NFR-UH1}
+  
+  NFR: UH-E1, UH-E3, UH-L1, UH-UP1, UH-UP2, OE-P1
+  
+  Type: Manual, User Test Demo
+            
+  Initial State: Application on home page, ready for use
+            
+  Input/Condition: User engagement
+            
+  Output/Result: Subjective feedback for how well application scores on different requirements relating to user experience
+            
+  How test will be performed: 
+  \begin{enumerate}
+    \item A user will use the application, going through the onboarding process
+    and doing a demo of the application
+    \item After a full demo, the user will be asked to provide feedback on
+    different aspects of usability, such as ease of use, app clarity, and prompt
+    politeness.
```

Comment:
Can we specify what they're being asked? We should put the exact wording of the question such as "On a scale of 1 to 10, how would you rate X". Then the output/result section can be "User feedback scores in categories X, Y, Z on a scale of 1 to 10"

Reply:
Include one question for each requirement that this test is for

Reply:
This belongs in a survey we have yet to make

Reply:
It needs to be *specific*. This document is meant to include that survey if there is one

Reply:
![{39F50886-B2B5-4A13-963B-719318B4FA75}](https://github.com/user-attachments/assets/71ff3f3f-3b61-4363-93ac-c3f5e485db5d)